packages/api-providers/src/polkadot/vanchor-deposit.ts

```diff
@@ -103,18 +105,26 @@ export class PolkadotVAnchorDeposit extends VAnchorDeposit<WebbPolkadot, Deposit
 
   async deposit(depositPayload: DepositPayload): Promise<VAnchorDepositResults> {
```

Comment:
I don't think the relayer flow should be added for deposit. The only time we should interact with a relayer in deposit is to fetch leaves (which we don't do for substrate - only web3.